### ISSUE TITLE
[Feature Request] Particle System Limiter

### ISSUE THREAD

**Issue by Chayotea** [pypy-vrc/VRCX#265]
https://docs.vrchat.com/docs/avatar-particle-system-limits

It is an old beta feature that sill works as I currently use it to shutdown users using particle lag and crashers. I used to simply never have lights and particles off till I found this.  

An issue is that VRCX does not like like me having it "Invalid JSON in config.json". I have already created a config.json from a long time ago with custom settings for most options that VRCX lets you add. Currently due to me having the particle limit added I am unable to use the VRCX  "VRChat Cache Management" or manually download worlds to cache. I have tested that simply me deleting all the lines that were the particle limit resulted in a valid config.json to use with VRCX.

**Comment by Natsumi-sama** [comment]
Can you send the lines you have added and copy your entire JSON file into this website https://jsonlint.com/ VRChat is more lenient about that file being valid, you likey have or are missing and extra comma or quote.

**Comment by Chayotea** [comment]
Yep fixed it just a comma on the last line lol I remade this multiple times and originally had the particle limit on the bottom. It is valid now.
{
	"betas": [
		"particle_system_limiter"
	],
	"ps_max_particles": 50000,
	"ps_max_systems": 200,
	"ps_max_emission": 5000,
	"ps_max_total_emission": 40000,
	"ps_mesh_particle_divider": 60,
	"ps_mesh_particle_poly_limit": 50000,
	"ps_collision_penalty_high": 50,
	"ps_collision_penalty_med": 30,
	"ps_collision_penalty_low": 10,
	"ps_trails_penalty": 10,
	"dynamic_bone_max_affected_transform_count": 64,
	"dynamic_bone_max_collider_check_count": 16,
	"cache_directory": "F:/VRCCache",
	"cache_size": 240,
	"cache_expiry_delay": 90
}

**Comment by GenesisAria** [comment]
Keep in mind, that the presence of the particle limiter lines at all completely disables the PlayerLocal layer, making it so that you will be unable to see player-interactive avatar effects.  It says so in the documentation only after i and maybe others, harassed the vrchat team back in 2018 that it was doing this without being specified.
